### PR TITLE
1.15.2

### PR DESCRIPTION
**Pull request by ericgaspar**
-Apply example_ynh

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: conf/.env.mysql [hunk 1]
```diff
@@ -13,7 +13,7 @@ APP_SECRET=__RANDOM_KEY__
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data/kimai.sqlite"
 # Configure your db driver and server_version in config/packages/doctrine.yaml
-DATABASE_URL=mysql://__DB_USER__:__DB_PASSWORD__@127.0.0.1:3306/__DB_NAME__
+DATABASE_URL=mysql://__DB_NAME__:__DB_PWD__@127.0.0.1:3306/__DB_NAME__
```
Comment (kevinpapst) [inline conf/.env.mysql:16]:
```suggestion
DATABASE_URL=mysql://__DB_USER__:__DB_PWD__@127.0.0.1:3306/__DB_NAME__
```

File: conf/.env.sqlite [hunk 2]
```diff
@@ -13,7 +13,7 @@ APP_SECRET=__RANDOM_KEY__
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data/kimai.sqlite"
 # Configure your db driver and server_version in config/packages/doctrine.yaml
-# DATABASE_URL=mysql://__DB_USER__:__DB_PASSWORD__@127.0.0.1:3306/__DB_NAME__
+# DATABASE_URL=mysql://__DB_NAME__:__DB_PWD__@127.0.0.1:3306/__DB_NAME__
```
Comment (kevinpapst) [inline conf/.env.sqlite:16]:
```suggestion
# DATABASE_URL=mysql://__DB_USER__:__DB_PWD__@127.0.0.1:3306/__DB_NAME__
```

**Review by kevinpapst — CHANGES_REQUESTED** [review]
Thanks for taking over and updating to latest version! 
I think there is a small copy&paste bug

File: conf/.env.mysql (hunk 1 above)
Comment (ericgaspar) [reply conf/.env.mysql:16]: Thanks for the feedback, this is not really a issue as `db_user=$db_name`

Comment (kevinpapst) [reply conf/.env.mysql:16]: Interesting detail about YH that I didn't know. Thanks for the explanation. Still looks like a bug though 😁

Comment (ericgaspar) [reply conf/.env.mysql]: I will use the `ynh_add_config`helper that replace those value automatically But for now you are right It is a bug. 👍

**Comment by ericgaspar** [comment]
!testme

**Review by yalh76 — CHANGES_REQUESTED** [review]
Most of the changes made here are not for an upgrade to 1.15.2 and I think, should be done on a dedicated PR

**Pull request closed without merging**